easybuild/easyconfigs/i/igraph/igraph-0.7.1-intel-2016b.eb

```diff
@@ -0,0 +1,31 @@
+easyblock = 'ConfigureMake'
+
+name = 'igraph'
+version = '0.7.1'
+
+homepage = 'http://igraph.org'
+description = """igraph is a collection of network analysis tools with the emphasis on 
+efficiency, portability and ease of use. igraph is open source and free. igraph can be 
+programmed in R, Python and C/C++."""
+
+toolchain = {'name': 'intel', 'version': '2016b'}
+toolchainopts = {'pic': True}
+
+sources = ['igraph-%(version)s.tar.gz']
+source_urls = ['https://github.com/igraph/igraph/releases/download/%(version)s']
+
+# Fix llapack and llblas when compiling with MKL
+# Fix aclocal which is hard-coded to version 1.14
+patches = ['igraph-0.7.1-no-lapack-no-blas.patch', 'igraph-0.7.1-fix-aclocal-version.patch']
```

Comment:
@mboisson please put each patch on it's own line, use `%(version)s` to avoid hardcoding the `igraph` version in multiple places, and include any descriptive info on the patches at the top of the patch itself, together with info on where the patch came from or who the author is (+ contact info)

```python
patches = [
    'igraph-%(version)s-no-lapack-no-blas.patch',
    'igraph-%(version)s-fix-aclocal-version.patch',
]
```